dev/release/03-binary-submit.sh

```diff
@@ -29,6 +29,7 @@ rc_number=$2
 version_with_rc="${version}-rc${rc_number}"
```

Comment:
Sorry about that, I think this is coming from moving the command between steps 03 to 02 when I was doing it initially. Should we change the `rc_number` var name here to `rc` to be more consistent between scripts?

Reply:
OK. I've changed to use `rc` here.